admin/src/components/LeftMenuHeader/Wrapper.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Logo from '../../assets/images/user-story-logo.svg';
+
+const Wrapper = styled.div`
```

Comment:
can we not just import the SCSS? I don't like the scalability limitations this brings with having embedded CSS in JS files